cli/cmd/testdata/install_output.golden

```diff
@@ -117,101 +117,164 @@ spec:
     targetPort: 8086
 
 ---
-kind: Deployment
 apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
I filed #372 regarding the fact that `conduit inject` perturbs the input unnecessarily like this. No action is needed to undo this in this PR.